tree/dataframe/inc/ROOT/RDF/RSampleInfo.hxx

```diff
@@ -36,11 +36,11 @@ class RSampleInfo {
    std::string fID;
    std::pair<ULong64_t, ULong64_t> fEntryRange;
 
-   ROOT::RDF::Experimental::RDatasetGroup fDatasetGroup;
+   ROOT::RDF::Experimental::RDatasetGroup fDatasetGroup{"", "", ""};
```

Comment:
@eguiraud I am unsure if I should get rid of this. Otherwise, I should introduce a default ctor for RDatasetGroup. Or is it okay like that?

Reply:
I don't think this is needed, `fDatasetGroup` is initialized at line 44